products/views.py

```diff
@@ -1,3 +1,61 @@
-from django.shortcuts import render
+import re
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Count
+
+from .models          import (
+    Category,
+    CategoryProduct,
+    Product,
+    ProductGroup,
+    ProductGroupImage,
+    ProductGroupPackageType,
+    Relation
+)
+
+class CategoryView(View):
```

Comment:
카테고리 리스트를 반환하는 엔드포인트라면 아래와 같이 수정해주세요~
```suggestion
class CategoriesView(View):
```